doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -32,21 +32,25 @@ datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_files = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)
```

Comment:
Was removing the doctest skip intentional? This will trigger a download of the data while doctesting, which is probably something that we want to avoid.